### ISSUE TITLE
ubuntu 18.04: cubic fails with  dbind-WARNING **: 21:08:00.213: Couldn't register with accessibility bus:

### ISSUE THREAD

**Issue by karawitan** [PJ-Singh-001/Cubic#72]
Hello,

  I just installed a fresh Ubuntu, did the system upgrades + install ssh and vncserver.
When I launch cubic, here is the message I get. What is missing here ?

kalou@majapatoche:~$ cubic

(cubic:15067): dbind-WARNING **: 21:08:00.213: Couldn't register with accessibility bus: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.

kalou@majapatoche:~$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.6 LTS"

kalou@majapatoche:~$ cubic -v

(cubic:15309): dbind-WARNING **: 21:18:58.966: Couldn't register with accessibility bus: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.

Cubic - Custom Ubuntu ISO Creator                                               

Traceback (most recent call last):
  File "/usr/bin/cubic", line 123, in <module>
    model.application.directory = os.path.dirname(os.path.realpath(__file__))
  File "/usr/share/cubic/cubic/utilities/fields.py", line 81, in __setattr__
    logger.log_value(f'Set {self.name} {display_key}', value)
  File "/usr/share/cubic/cubic/utilities/logger.py", line 82, in log_value
    if verbose: _log_value_hanging(column_a_text, column_b_text, column_a_initial_indent='  \u2022 ', column_a_subsequent_indent='    ')
  File "/usr/share/cubic/cubic/utilities/logger.py", line 254, in _log_value_hanging
    print(f'{column_a_line}{column_b_line}')
UnicodeEncodeError: 'ascii' codec can't encode character '\u2022' in position 2: ordinal not in range(128)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/cubic", line 304, in <module>
    logger.log_value('Exception', exception)
  File "/usr/share/cubic/cubic/utilities/logger.py", line 82, in log_value
    if verbose: _log_value_hanging(column_a_text, column_b_text, column_a_initial_indent='  \u2022 ', column_a_subsequent_indent='    ')
  File "/usr/share/cubic/cubic/utilities/logger.py", line 254, in _log_value_hanging
    print(f'{column_a_line}{column_b_line}')
UnicodeEncodeError: 'ascii' codec can't encode character '\u2022' in position 2: ordinal not in range(128)

**Comment by karawitan** [comment]
Ok this happens only through vnc.
Linux box is not my primary workstation, so I'm using cubic through vnc.
Launching cubic on the same not using vnc works. Weird :)

**Comment by karawitan** [comment]
Ok found the fix !

export  LC_ALL="en_US.UTF-8"
